app/src/main/java/ar/com/wolox/android/bootstrap/ui/posts/PostsFragment.kt

```diff
@@ -4,65 +4,104 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
 import androidx.fragment.app.viewModels
+import androidx.navigation.fragment.findNavController
 import androidx.recyclerview.widget.LinearLayoutManager
 import ar.com.wolox.android.bootstrap.Constants.INTERNAL_SERVER_ERROR_STATUS_CODE
 import ar.com.wolox.android.bootstrap.Constants.NOT_FOUND_STATUS_CODE
 import ar.com.wolox.android.bootstrap.R
 import ar.com.wolox.android.bootstrap.databinding.FragmentPostsBinding
 import ar.com.wolox.android.bootstrap.network.util.RequestStatus
 import ar.com.wolox.android.bootstrap.ui.adapter.PostsAdapter
-import ar.com.wolox.android.bootstrap.ui.base.BaseFragment
+import ar.com.wolox.android.bootstrap.ui.root.RootViewModel
 import ar.com.wolox.android.bootstrap.utils.SnackbarFactory
 import dagger.hilt.android.AndroidEntryPoint
 
 @AndroidEntryPoint
-class PostsFragment : BaseFragment<PostsView, PostsViewModel>(), PostsView {
+class PostsFragment : Fragment() {
 
-    private lateinit var binding: FragmentPostsBinding
+    private var _binding: FragmentPostsBinding? = null
 
-    override val viewModel: PostsViewModel by viewModels()
+    private val binding get() = _binding!!
+
+    val viewModel: PostsViewModel by viewModels()
+    private val rootViewModel: RootViewModel by activityViewModels()
 
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
-        binding = FragmentPostsBinding.inflate(inflater)
-        getPosts()
+        _binding = FragmentPostsBinding.inflate(inflater)
         return binding.root
     }
 
-    override fun showErrorSnackbar() {
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        if (viewModel.isUserLogged) {
+            // TODO: Is there a better way to always show the actionbar in a particular fragment?
```

Comment:
It's not that simple 😆 However, why do you want to hide the action bar if the user is not logged?

Reply:
Hiding the action bar in the login screen is the behavior the Bootstrap already had. I am just trying to keep that.

Reply:
Well, that was not expected to happen 😆 Good catch!